packages/nimble-components/src/wafer-map/specs/README.md

```diff
@@ -99,6 +99,7 @@ _The key elements of the component's public API surface:_
     -   `colorScaleMode` - represent an Enum value that determent if the colorScale is represent a continues gradient values (linear), or is set categorically (ordinal).
     -   `highlightedValues` - represent a list of strings of dies values that will be highlighted in the wafer map view
     -   disabled - it's represented by a boolean value and refers to the state of the `nimble-wafer-map` component. If true, the component should be rendered dimmed out and no user interaction should be allowed
+    -   `diePadding` - a value between 0 and 1, defaulted at 0, representing the percentage of the individual die space taken by padding.
```

Comment:
What happens if a Number value is given outside the range of 0-1? (-Inifinty, Infinity, NaN, -5, 5) Falls back to the default value of zero? Make sure that is covered by the spec:
```suggestion
    -   `diePadding` - a number value between 0 and 1, where all other values fallback to the default of 0, representing the percentage of the individual die space taken by padding.
```